### PR TITLE
fix(github): update link to the project page in README.md

### PR DESCRIPTION
**Pull request by batazor**
update link to a documentation

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#versioning)
* [x] I have updated the documentation according to [documentation](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#documentation)
* [x] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog).
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/blob/master/community/CONTRIBUTING.md).
* [x] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/stable/developer-guide/ci/)).

Changes are automatically published when merged to `main`. They are not published on branches.

### CODE REVIEW

**Comment by batazor** [comment]
> Thanks, could you also please bump the version/changelog and rebase your branch?

I ended up not understanding the `bump the version/changelog`

we're not changing any chart as such

**Comment by batazor** [comment]
> Thanks, could you also please bump the version/changelog and rebase your branch?

done

File: charts/argo-cd/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: v2.4.4
 description: A Helm chart for Argo CD, a declarative, GitOps continuous delivery tool for Kubernetes.
 name: argo-cd
-version: 4.9.12
+version: 4.9.13
```
Comment (mkilchhofer) [inline charts/argo-cd/Chart.yaml:5]:
```suggestion
version: 4.9.12
```

File: charts/argo-cd/Chart.yaml [hunk 2]
```diff
@@ -21,4 +21,4 @@ dependencies:
     condition: redis-ha.enabled
 annotations:
   artifacthub.io/changes: |
-    - [Changed]: Update to app version 2.4.4"
+    - [Changed]: Update link to Argo CD website
```
Comment (mkilchhofer) [inline charts/argo-cd/Chart.yaml:24]:
```suggestion
    - [Changed]: Update to app version 2.4.4"
```

**Review by mkilchhofer — CHANGES_REQUESTED** [review]
As it is not related to a helm chart lets restore Chart.yaml of argo-cd chart

**Review by mkilchhofer — APPROVED** [review]

**Pull request merged**